skore-ui/src/components/TreeAccordion.vue

```diff
@@ -23,13 +22,9 @@ const props = defineProps<{
 const emit = defineEmits<{
   itemAction: [action: string, itemName: string];
 }>();
-const lastItemAction = ref<string | null>(null);
-const lastItemName = ref<string | null>(null);
 
-watch([lastItemAction, lastItemName], () => {
-  if (lastItemAction.value && lastItemName.value) {
-    emit("itemAction", lastItemAction.value, lastItemName.value);
-  }
```

Comment:
Out of curiosity, do you remember why this condition was there in the first place?

Reply:
Ref's value can be null hence we check to please the compiler

Reply:
Ah I see; I misread this and thought that we were checking whether the same item was clicked twice in a row, for some reason ^^'